/android_slave/workspace/ccs-pr/node/android/build/../cocos/ui/UIEditBox/UIEditBoxImpl-android.cpp:88:48: error: no matching function for call to 'cocos2d::Label::setColor(cocos2d::Color4B&)'

--- a/cocos/ui/UIEditBox/UIEditBoxImpl-android.cpp
+++ b/cocos/ui/UIEditBox/UIEditBoxImpl-android.cpp
@@ -76,7 +76,7 @@ bool EditBoxImplAndroid::initWithSize(const Size& size)
 	// align the text vertically center
     _label->setAnchorPoint(Vec2(0, 0.5f));
     _label->setPosition(Vec2(CC_EDIT_BOX_PADDING, size.height / 2.0f));
-    _label->setColor(_colText);
+    _label->setTextColor(_colText);
     _editBox->addChild(_label);
 	
     _labelPlaceHolder = Label::create();
@@ -85,7 +85,7 @@ bool EditBoxImplAndroid::initWithSize(const Size& size)
     _labelPlaceHolder->setAnchorPoint(Vec2(0, 0.5f));
     _labelPlaceHolder->setPosition(CC_EDIT_BOX_PADDING, size.height / 2.0f);
     _labelPlaceHolder->setVisible(false);
-    _labelPlaceHolder->setColor(_colPlaceHolder);
+    _labelPlaceHolder->setTextColor(_colPlaceHolder);
     _editBox->addChild(_labelPlaceHolder);
     
     _editSize = size;

--- a/cocos/ui/UIEditBox/UIEditBoxImpl-win32.cpp
+++ b/cocos/ui/UIEditBox/UIEditBoxImpl-win32.cpp
@@ -527,7 +527,7 @@ bool EditBoxImplWin::initWithSize(const Size& size)
 	// align the text vertically center
     _label->setAnchorPoint(Vec2(0, 0.5f));
     _label->setPosition(Vec2(CC_EDIT_BOX_PADDING, size.height / 2.0f));
-    _label->setColor(_colText);
+    _label->setTextColor(_colText);
     _editBox->addChild(_label);
 
     _labelPlaceHolder = Label::create();
@@ -536,7 +536,7 @@ bool EditBoxImplWin::initWithSize(const Size& size)
     _labelPlaceHolder->setAnchorPoint(Vec2(0, 0.5f));
     _labelPlaceHolder->setPosition(CC_EDIT_BOX_PADDING, size.height / 2.0f);
     _labelPlaceHolder->setVisible(false);
-    _labelPlaceHolder->setColor(_colPlaceHolder);
+    _labelPlaceHolder->setTextColor(_colPlaceHolder);
     _editBox->addChild(_labelPlaceHolder);
     
     _editSize = size;

--- a/cocos/ui/UIEditBox/UIEditBoxImpl-wp8.cpp
+++ b/cocos/ui/UIEditBox/UIEditBoxImpl-wp8.cpp
@@ -105,7 +105,7 @@ bool UIEditBoxImplWp8::initWithSize( const Size& size )
 	// align the text vertically center
 	m_pLabel->setAnchorPoint(Vec2(0.0f, 0.5f));
 	m_pLabel->setPosition(Vec2(5.0, size.height / 2.0f));
-	m_pLabel->setColor(m_colText);
+	m_pLabel->setTextColor(m_colText);
 	_editBox->addChild(m_pLabel);
 
 	m_pLabelPlaceHolder = Label::createWithSystemFont("", "", size.height-12);
@@ -113,7 +113,7 @@ bool UIEditBoxImplWp8::initWithSize( const Size& size )
 	m_pLabelPlaceHolder->setAnchorPoint(Vec2(0.0f, 0.5f));
 	m_pLabelPlaceHolder->setPosition(Vec2(5.0f, size.height / 2.0f));
 	m_pLabelPlaceHolder->setVisible(false);
-	m_pLabelPlaceHolder->setColor(m_colPlaceHolder);
+	m_pLabelPlaceHolder->setTextColor(m_colPlaceHolder);
 	_editBox->addChild(m_pLabelPlaceHolder);
 
 	m_EditSize = size;

